packages/blade/src/components/Form/SelectorSupportText.tsx

```diff
@@ -0,0 +1,28 @@
+import { BaseText } from '~components/Typography/BaseText';
+import { getPlatformType } from '~utils';
+
+const SelectorSupportText = ({
+  children,
+  id,
+}: {
+  children: React.ReactNode;
+  id?: string;
+}): React.ReactElement => {
+  const isReactNative = getPlatformType() === 'react-native';
+
+  return (
+    <BaseText
+      id={id}
+      as={isReactNative ? undefined : 'span'}
+      color="surface.text.muted.lowContrast"
```

Comment:
color doesn't changes when the checkbox is disabled?

Reply:
Yeah support text doesn't have any changes.